src/components/Nav.jsx

```diff
@@ -30,7 +30,11 @@ export default function Nav() {
           <TabList onChange={handleChange} centered>
             <Tab
               label={
-                <Link className="links" to="/">
+                <Link
+                  className="links"
+                  to="/"
+                  onClick={() => (window.location.href = "/")}
+                >
```

Comment:
the `to` prop should do the job here!

the `onClick` is a nice workaround but shouldn't be needed here